app/lib/efile/dependent_eligibility/qualifying_relative.rb

```diff
@@ -37,6 +37,13 @@ def requires_member_of_household_test?
 
       private
 
+      def filer_financially_supported?
+        # this is not defined or required on archived dependents
```

Comment:
nitty, but because we changed the name of the method "this" is a little murky in terms of what it refers to.